exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -629,6 +629,7 @@ drill.exec.options: {
     store.parquet.reader.columnreader.async: false,
     store.parquet.reader.int96_as_timestamp: false,
     store.parquet.reader.strings_signed_min_max: "",
+    store.parquet.refresh_timeout_per_runnable_in_msec: 15000,
```

Comment:
See prior comment about omitting 'refresh' .  Also I think that we should not put this in the 'parquet' namespace even though currently it is mainly used for Parquet metadata read.  The reason is the runnable timeout could in theory be used for other formats also.  How about `store.timeout_per_runnable_in_msec`  ?